### DISCUSSION TITLE
Link to wiki in UI + Prompt / region checker

### DISCUSSION THREAD

**Discussion by MrUrzza** [hako-mikan/sd-webui-regional-prompter#151] [Ideas]
As the title states, a link to a guide, wiki, or this git page in the UI its self would be nice. This is a fairly complex extension, and having the manual on quick tap would be a huge help.

Another idea is a button that when pressed, instead of generating an image it would instead show you what regions are going to be effected by what part of your prompt.

**Comment by Symbiomatrix** [comment]
I'm game, as long as it's not too much effort & distracting. You have any guides / wiki in mind? Where & how exactly would you link it in the ui?
There is such a button for matrix mode, and prompt mode (optionally) shows the masks it used for each keyword along with the generation. I could probably mimic hako's visualisation in inpaint+'s mask display thing (though the regions can be segmented freely so it's less clear cut), but I still have more pressing usability features slated for it first.
https://github.com/hako-mikan/sd-webui-regional-prompter#visualise-and-make-template

**Reply by MrUrzza** [reply]
You could probably just use the main page for this extension for the wiki. I'm thinking something like this:
![Link UI](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/134964094/cb4f599d-92a6-4c26-a266-35826d375e34)

**Reply by Symbiomatrix** [reply]
Gradio is based on components; Per the docs, there's one for adding arbitrary html, I guess I could use that. Do you happen to know of anywhere in the webui or an extension that shows such a link?

**Reply by MrUrzza** [reply]
Dynamic prompts has links to their github in "app" under the "Need Help?" tab. It's where I got the idea from.
![Links](https://github.com/hako-mikan/sd-webui-regional-prompter/assets/134964094/4a498e90-f107-472a-8560-cba9173d4f11)

**Reply by Symbiomatrix** [reply]
Thanks. Yep, they're using an html component, displaying a substituted file (helptext.html).

**Comment by Symbiomatrix** [comment]
It is done.
